ote/src/cljs/ote/app/controller/transit_visualization.cljs

```diff
@@ -183,31 +183,35 @@
   (days-to-first-diff start-date date->hash))
 
 (define-event LoadServiceChangesForDateResponse [response detection-date]
-  {:path [:transit-visualization]}
-  (assoc app
-    :service-changes-for-date-loading? false
-    :service-info (:service-info response)
-    :changes-all (sort-by :different-week-date < (:route-changes response))
-    :changes-route-no-change (sorted-route-changes true (future-changes detection-date (:route-changes response)))
-    :changes-route-filtered (sorted-route-changes false (future-changes detection-date (:route-changes response)))
-    :gtfs-package-info (:gtfs-package-info response)
-    :route-hash-id-type (:route-hash-id-type response)))
-
-(defn- init-view-state [app]
+              {:path [:transit-visualization]}
```

Comment:
we have treated define-event macro as function so that next line indention is 2 spaces

Reply:
Fix'd